### DISCUSSION TITLE
TuringCourses/major/digital_image_processing/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#30] [General]
# TuringCourses/major/digital_image_processing/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major/digital_image_processing/

**Comment by HobbitQia** [comment]
期末基本上是文科题，考默写（确信）复习要好好看PPT。
PPT下载下来下面有中文注释，记得及时下载，不然可能会过期。

**Reply by HobbitQia** [reply]
贴一个 2022Fall 的[资源仓库](https://github.com/HobbitQia/ZJU-Courses-Resources/tree/master/22fa/%E5%9B%BE%E5%83%8F%E4%BF%A1%E6%81%AF%E5%A4%84%E7%90%86(DIP))，里面有六次实验的实验报告，仅供参考。

**Comment by smdrr11037** [comment]
[菜鸡的作业心得](https://github.com/smdrr11037/ZJU-Course/blob/main/dip_hw.md)
